### PR TITLE
[SecurityBundle] Deprecate ACL related code

### PR DESCRIPTION
**Pull request by chalasr**
| Q             | A
| ------------- | ---
| Branch?       | 3.4
| Bug fix?      | no
| New feature?  | yes
| BC breaks?    | no
| Deprecations? | yes
| Tests pass?   | yes/no
| Fixed tickets | replaces #23811
| License       | MIT
| Doc PR        | todo

Needs https://github.com/symfony/acl-bundle/pull/2

### CODE REVIEW

File: src/Symfony/Bundle/SecurityBundle/Command/InitAclCommand.php [hunk 1]
```diff
@@ -93,7 +89,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        // BC to be removed in 4.0
+        (new SymfonyStyle($input, $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output))->warning('Command "init:acl" is deprecated since version 3.4 and will be removed from SecurityBundle in 4.0. Register symfony/acl-bundle and use "acl:init" instead.');
```
Comment (javiereguiluz) [inline src/Symfony/Bundle/SecurityBundle/Command/InitAclCommand.php]: `Register symfony/acl-bundle and use ...` -> `Install symfony/acl-bundle and use ...` ?

File: src/Symfony/Bundle/SecurityBundle/Command/SetAclCommand.php [hunk 2]
```diff
@@ -117,7 +117,8 @@ protected function configure()
      */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
-        // BC to be removed in 4.0
+        (new SymfonyStyle($input, $output instanceof ConsoleOutputInterface ? $output->getErrorOutput() : $output))->warning('Command "acl:set" is deprecated since version 3.4 and will be removed from SecurityBundle in 4.0. Register symfony/acl-bundle instead.');
```
Comment (javiereguiluz) [inline src/Symfony/Bundle/SecurityBundle/Command/SetAclCommand.php]: `Register symfony/acl-bundle instead.` -> `Install symfony/acl-bundle to use this command.` ?

**Review by javiereguiluz — APPROVED** [review]

**Review by ro0NL — COMMENTED** [review]
Thanks! Looking good :)

**Review by gusdecool — APPROVED** [review]

**Review by nicolas-grekas — APPROVED** [review]

**Review by weaverryan — APPROVED** [review]

**Comment by fabpot** [comment]
Thank you @chalasr.

**Pull request merged**